src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -191,6 +222,53 @@ export default function GamesSubmenu({
                 {t('submenu.addShortcut', 'Add shortcut')}
               </button>
             )}
+            {isLinux && !eosOverlayInstalling && (
+              <button
+                className="link button is-text is-link"
+                onClick={async () => {
+                  const { winePrefix } = await ipcRenderer.invoke(
+                    'requestSettings',
+                    appName
+                  )
+                  if (eosOverlayEnabled) {
+                    await ipcRenderer.invoke('disableEosOverlay', winePrefix)
+                    setEosOverlayEnabled(false)
+                  } else {
+                    const initialEnableResult = await ipcRenderer.invoke(
+                      'enableEosOverlay',
+                      winePrefix
+                    )
+                    const { installNow } = initialEnableResult
+                    let { wasEnabled } = initialEnableResult
+
+                    if (installNow) {
+                      await handleGameStatus({
+                        appName: eosOverlayAppName,
+                        runner: 'legendary',
+                        status: 'installing'
+                      })
+                      setEosOverlayInstalling(true)
+                      await ipcRenderer.invoke('installEosOverlay')
+                      await handleGameStatus({
+                        appName: eosOverlayAppName,
+                        runner: 'legendary',
+                        status: 'done'
+                      })
+                      setEosOverlayInstalling(false)
```

Comment:
Ideally this would be a helper function somewhere, I'm just not quite sure how to create those here